packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js

```diff
@@ -245,6 +251,12 @@ export const TagsDescription = () => {
                 }) }
             </TextListItem>
         </TextList>
+        <Text component={ TextVariants.p }>
+            { intl.formatMessage({
+                id: 'cost.tags.aliases_org_units',
```

Comment:
nitpick: I am using camelCase for these IDs.. but it doesn't matter too much

Reply:
Done :+1: